app/utils.py

```diff
@@ -7,13 +8,13 @@
     LetterPrintTemplate,
     SMSMessageTemplate,
 )
-from notifications_utils.timezones import convert_bst_to_utc
+from notifications_utils.timezones import convert_local_timezone_to_utc
 from sqlalchemy import func
 
 DATETIME_FORMAT_NO_TIMEZONE = "%Y-%m-%d %H:%M:%S.%f"
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 DATE_FORMAT = "%Y-%m-%d"
-local_timezone = pytz.timezone("Europe/London")
+local_timezone = pytz.timezone(getenv("TIMEZONE", "America/New_York"))
```

Comment:
[suggestion] (and this occurs a couple other places in the code) each time we're doing this, we could actually import it from `notifications_utils.timezones`, so line 11 would become:

`from notifications_utils.timezones import convert_local_timezone_to_utc, local_timezone`

I think that would really clarify that the `local_timezone` variable and the local timezone referenced in the method name are the same.